src/Layouts/stars/feed/starDesc.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+  TextField,
+  Typography,
+  FormControl,
+  InputLabel,
+  Select,
+  Chip,
+  MenuItem,
+  Input,
+} from "@mui/material";
+import { starType, resources } from "../../../assets/star";
+import { useState } from "react";
+
+interface starProps {
+  star: starType;
+}
+
+const StarDesc = ({ star }: starProps) => {
+  const [resourceList, setResourceList] = useState(star.resources);
+
+  return (
+    <div className="starDesc">
+      <div className="header">
+        <h1>
+          <span
+            id="priority"
+            style={{
+              color:
+                star.severity === 1
```

Comment:
const colors = {
  1: "red",
  2: "orange",
  3: "yellow",
};

 colors[star.severity] ||  "green"

Reply:
fixed in asnother way

Reply:
fixed in asnother way